config/manager/manager.yaml

```diff
@@ -53,5 +53,10 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        env:
```

Comment:
This will always set the WATCH_NAMESPACES to the place where you deploy the operator.
Or am I thinking wrong?

Reply:
It seems like it, but when I run `make bundle` with this, it actually sets the watch namespace to `metadata.annotations['olm.targetNamespaces']` in the CSV. But I'll change it here as well.